BitwardenShared/UI/Platform/Application/Views/LoadingOverlayView.swift

```diff
@@ -21,6 +21,7 @@ struct LoadingOverlayView: View {
         }
         .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .center)
         .background(Asset.Colors.backgroundSecondary.swiftUIColor.opacity(0.9).ignoresSafeArea())
+        .accessibilityIdentifier("LoadingViewImage")
```

Comment:
♻️ @ifernandezdiaz Maybe to update it in the future but not sure why `Image` was used here as a suffix when it's more than a simple image view.